projects/ngx-translate-router/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gilsdav/ngx-translate-router",
-  "version": "7.2.1",
+  "version": "7.2.2",
```

Comment:
Let’s update the version to 8.0.0, as the library now utilizes ngx-translate v16 and Angular v19.